antarest/study/storage/rawstudy/model/filesystem/lazy_node.py

```diff
@@ -148,7 +176,7 @@ def load(
         Args:
             url: data path to retrieve
             depth: after url is reached, node expand tree until matches depth asked
-            expanded: context parameter to determine if current node become from a expansion
+            expanded: context parameter to determine if current node become from an expansion
```

Comment:
Should be `comes` instead of `become` no ?

Reply:
You should ask the one who wrote it ... I guess you right, It should be even `comes` with an `s` 

I'mma correct this